src/settings.cpp

```diff
@@ -187,9 +187,8 @@ static int ParseIntList(const char *p, int *items, int maxitems)
 			default: {
 				if (n == maxitems) return -1; // we don't accept that many numbers
 				char *end;
-				long v = strtol(p, &end, 0);
+				size_t v = strtoul(p, &end, 0);
```

Comment:
surprised this doesn't cause a warning elsewhere (e.g. line192), given this has gone from signed to unsigned...

Reply:
Also, shouldn't this be `unsigned long` instead of `size_t` ? They're not necessarily the same...

Reply:
I think the entire SDT_INTLIST handling might just as well be changed to only do unsigned. As far as I can tell no uses of it need signed values anyway.